themes/gatsby-theme-notes/src/components/note.js

```diff
@@ -7,8 +7,9 @@ const WikiPage = ({
   data: {
     note: { body },
   },
+  ...props
 }) => (
-  <Layout>
+  <Layout {...props}>
```

Comment:
The blog header uses `props.location`, so there's a little bit of props drilling added